docs/sources/query/uql.md

```diff
@@ -76,7 +76,7 @@ parse-json
 
 ### project-away
 
-`project-away` command is exactly opposite as `project`. It just drops specific columns from the data. It doesn't support alias or dot notation selector.
+`project-away` command is exactly opposite as `project`. It just drops specific columns from the data. It does not support alias or dot notation selector.
```

Comment:
We prefer contractions: https://grafana.com/docs/writers-toolkit/write/style-guide/voice-tone-guidelines/#tone

```suggestion
`project-away` command is exactly opposite as `project`. It just drops specific columns from the data. It doesn't support alias or dot notation selector.
```